### PR TITLE
New. Integration. CoBlocks integration implemented.

### PR DESCRIPTION
**Pull request by Glomberg**
https://doboard.com/1/task/13168

### CODE REVIEW

File: inc/cleantalk-pluggable.php [hunk 1]
```diff
@@ -1439,11 +1439,19 @@ class_exists('Cleantalk\Antispam\Integrations\CleantalkInternalForms')
             return 'Plugin Name: WooCommerce Product Enquiry Premium - have the direct integration';
         }
 
-        // WP Discuz skip service requests. The plugin have the direct integration
+        // CoBlocks requests. The plugin have the direct integration
```
Comment (alexandergull) [inline inc/cleantalk-pluggable.php:1442]: Change this back to WP Discuz

File: lib/Cleantalk/Antispam/Integrations/CoBlocks.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+namespace Cleantalk\Antispam\Integrations;
+
+use Cleantalk\ApbctWP\Variables\Post;
+
+class CoBlocks extends IntegrationBase
+{
+    /**
+     * @inheritDoc
+     */
+    public function getDataForChecking($argument)
```
Comment (alexandergull) [inline lib/Cleantalk/Antispam/Integrations/CoBlocks.php:12]: What about event token? Does the form allow the custom token field transmission?

**Review by alexandergull — CHANGES_REQUESTED** [review]

File: inc/cleantalk-pluggable.php (hunk 1 above)
Comment (Glomberg) [reply inc/cleantalk-pluggable.php]: Thanks. Done https://github.com/CleanTalk/wordpress-antispam/pull/485/commits/67441afeac027de2211b5705fdc5e3424b08be5f

File: lib/Cleantalk/Antispam/Integrations/CoBlocks.php (hunk 2 above)
Comment (Glomberg) [reply lib/Cleantalk/Antispam/Integrations/CoBlocks.php:12]: Event token works without any additional movements ;-)

**Pull request merged**